examples/user_guide/tiling.ipynb

```diff
@@ -7,6 +7,16 @@
     "## Building Tilesets using Datashader"
    ]
   },
+  {
```

Comment:
I think we should move this back again. No need to show something which does not work.

Reply:
OK. I'll just skip it from the build and toctree.